database/seed.js

```diff
@@ -0,0 +1,39 @@
+const Promise = require('bluebird');
+const faker = require('faker');
+const schema = Promise.promisifyAll(require('./data/schemaHelpers'));
+
+// note default properties: id, timestamp, and isliked
+// const sampleSong = {
+//   length: Math.floor(Math.random() * (210 - 100) + 100),
+//   timestamp: 0,
+//   isliked: 0,
+//   songfile: faker.internet.url(),
+//   title: faker.random.words(),
+//   artist: faker.name.findName(),
+//   album: faker.commerce.productName(),
+//   thumbnail: faker.image.imageUrl(),
+// };
+
+// console.log(sampleSong);
+
+const seedData = () => {
+  for (let i = 0; i < 100; i += 1) {
+    let sampleSong = {
+      length: Math.floor(Math.random() * (210 - 100) + 100),
```

Comment:
This line is confusing to understand on two points:
  1. You are seemingly creating a random number between 210 and 100, but I don't really know why. Are these just arbitrary limits or are they coming from somewhere?
  2. Why is "(210-100)" written this way? Why not just use 110?
i.e.:
```suggestion
      length: Math.floor(Math.random() * 110 + 100),
```

Reply:
I wanted realistic song lengths--so I figured 100sec to 210 were reasonable.